Content.Shared/Animals/UdderComponent.cs

```diff
@@ -0,0 +1,56 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Animals;
+
+/// <summary>
+///     Gives the ability to produce a solution;
+///     produces endlessly if the owner does not have a HungerComponent.
+/// </summary>
+[RegisterComponent, AutoGenerateComponentPause, NetworkedComponent]
+public sealed partial class UdderComponent : Component
+{
+    /// <summary>
+    ///     The reagent to produce.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
```

Comment:
```suggestion
    [DataField, AutoNetworkedField]
```
should be fine to have this editable using VV for funny admemes :)
but it should be networked then